pkg/cloud/services/networking/port.go

```diff
@@ -551,9 +574,57 @@ func (s *Service) IsTrunkExtSupported() (trunknSupported bool, err error) {
 	return true, nil
 }
 
-// AdoptPorts looks for ports in desiredPorts which were previously created, and adds them to resources.Ports.
+// AdoptPortsMachine looks for ports in desiredPorts which were previously created, and adds them to resources.Ports.
+// A port matches if it has the same name and network ID as the desired port.
+func (s *Service) AdoptPortsMachine(scope *scope.WithLogger, desiredPorts []infrav1.ResolvedPortSpec, resources *infrav1.MachineResources) error {
```

Comment:
Same as above. Worth having just one of these? Or perhaps we delete one later in the series and it doesn't matter...

Reply:
Since we'll stop populating the OpenStackMachine status at some point, we won't need these things.